editor/plugins/item_list_editor_plugin.cpp

```diff
@@ -350,6 +357,11 @@ ItemListEditor::ItemListEditor() {
 
 	hbc->add_spacer();
 
+	clear_button = memnew(Button);
```

Comment:
Should the name in the code match the name given in the display? For consistenct?

`delete_all_button`

Reply:
Maybe. I thought it's easier to distinguish this way.